### PR TITLE
Simplify file copying

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

File: config/config-package.py [hunk 1]
```diff
@@ -177,70 +166,58 @@ def cfg_option(self, section, name, default=DEFAULT):
...
+        if destination is None:
+            if template_name.endswith('.j2'):
+                destination = self.path / template_name[:-3]  # remove `.j2`
+            else:
+                destination = self.path / template_name
+
         with open(destination, 'w') as f_:
             f_.write(content)
 
+        return destination.name
```
Comment (mauritsvanrees) [inline config/config-package.py:208]: This returns only the file name, not the path. This is wrong for `.github/workflows/linting.yml`. Instead we should do `destination.relative_to(self.path)`.

File: config/config-package.py [hunk 2]
```diff
@@ -177,70 +166,58 @@ def cfg_option(self, section, name, default=DEFAULT):
...
         self._add_project_to_config_type_list()
 
-        self.editorconfig()
-        self.lint_requirements()
-        self.linting_yml()
-        self.pyproject_toml()
-        self.setup_cfg()
-        self.tox()
+        files_changed = [
+            self.editorconfig(),
```
Comment (mauritsvanrees) [inline config/config-package.py:214]: The `files_changed` are all None, because the methods do not return the destination name/path.

**Review by mauritsvanrees — APPROVED** [review]
That is easier, thanks.
I tried it and it failed. I added two inline comments for this. I will push a commit to fix this.
Can you check again with my change?

**Comment by gforcada** [comment]
Thanks for the fixes, indeed, I did not test them before pushing, sorry 🙏🏾

**Pull request merged**